charts/psnrChartHandler.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This function is responsible for taking the json result
+ * from a psnr test and transforming it into a format
+ * to be fed into a highchart series field.  It will
+ * build a series where the x axis is the jenkins build
+ * number and the y axis is the psnr value for that build.
+ *
+ * @param json the json result data from the psnr test
+ *
+ * @returns an object suitable to be placed in a series array
+ * for a highchart
+ */
+export default json => {
```

Comment:
Why not break the fetch out as well, maybe into this helper?

Reply:
hmm but then the fetch logic would be duplicated for each chart